src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -100,7 +100,8 @@ public static class ScriptConstants
         public const string AntaresLogIdHeaderName = "X-ARR-LOG-ID";
         public const string AntaresScaleOutHeaderName = "X-FUNCTION-SCALEOUT";
         public const string AntaresColdStartHeaderName = "X-MS-COLDSTART";
-        public const string SiteTokenHeaderName = "x-ms-site-restricted-token";
+        public const string SiteRestrictedTokenHeaderName = "x-ms-site-restricted-token";
+        public const string SiteTokenHeaderName = "x-ms-site-token";
```

Comment:
Curious how we decided on this header name? Are there existing examples of this header name being used? If not, are there any examples out there of an alt-auth header that we should consider re-using?

If we arbitrarily decided on this name and don't think it is worth spending much time on picking the best name I am fine with that - so long as we at least acknowledge this is the route we are choosing.